_health-care/_js/routes.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Route } from 'react-router';
+
+import AdditionalInformationSection from './components/personal-information/AdditionalInformationSection';
+import AdditionalMilitaryInformationSection from './components/military-service/AdditionalMilitaryInformationSection';
+import AnnualIncomeSection from './components/financial-assessment/AnnualIncomeSection';
+import ChildInformationSection from './components/financial-assessment/ChildInformationSection';
+import DeductibleExpensesSection from './components/financial-assessment/DeductibleExpensesSection';
+import DemographicInformationSection from './components/personal-information/DemographicInformationSection';
+import FinancialDisclosureSection from './components/financial-assessment/FinancialDisclosureSection';
+import InsuranceInformationSection from './components/insurance-information/InsuranceInformationSection';
+import IntroductionSection from './components/IntroductionSection.jsx';
+import MedicareMedicaidSection from './components/insurance-information/MedicareMedicaidSection';
+import NameAndGeneralInfoSection from './components/personal-information/NameAndGeneralInfoSection';
+import ReviewAndSubmitSection from './components/ReviewAndSubmitSection.jsx';
+import ServiceInformationSection from './components/military-service/ServiceInformationSection';
+import SpouseInformationSection from './components/financial-assessment/SpouseInformationSection';
+import VAInformationSection from './components/personal-information/VAInformationSection';
+import VeteranAddressSection from './components/personal-information/VeteranAddressSection';
+
+const routes = [
+  // Personal Information routes.
+  <Route
+      component={AdditionalInformationSection}
+      key="/personal-information/additional-information"
```

Comment:
ugh. My current understanding (this has morphed multiple times in the past week so take with a grain of salt) is that React internally actually generates a "key" for each Virtual Dom element that it uses re-identify the component when it does a re-render.  If it can tell it's the same component, it won't destroy/recreate it thereby reducing thrashing.  This is a large part of react's performance.

When you create a "dynamic" component...one where you are storing components in a Javascript construct like an array and then programmatically adding them to another, React prefers you manually specify a key that lets it identify the component.  I *think* key is conceptually the `id` of the component...but since `id` has special meaning in HTML/XML, they had to give it another name.

You could imagine if this was a dynamic list (like rich AJAX search results) where the items got reordered, you'd want to avoid rerendering those items...in fact you'd only want to render the new items. By default, the key is the array index so on a reorder, react wouldn't be able to tell that it was actually reusing components and would rerender everything. If you put in a unique semantic key, then react can sort things out.

Of course, in this case, the list is static so the code is mainly to satisfy the OCD linter that can't tell between an Javascript array that is being used immutably and a dynamically generated one...but whatever.

Tagging @crwallace in case she a better understanding.